dashboard-parent/dashboard-core/pom.xml

```diff
@@ -31,5 +32,33 @@
 			<artifactId>xstream</artifactId>
 			<version>${xstream.version}</version>
 		</dependency>
+		
```

Comment:
All this changes are not required, defined in parent pom

Reply:
I tried to use as minimal dependencies as possible.

Reply:
Actually jetty is usually being added to the `examples` sub-project: https://github.com/wicketstuff/core/blob/master/dashboard-parent/dashboard-examples/pom.xml
This keeps main library clean and small
Are you sure this is required here?

Reply:
Do i need to implement my tests in example project? I like to provide tests that guarantee my usecase works. But then i need junit and with wicket tester i need servlet library. So far i always used jetty to provide this. Do you have another example how i can test in core library without using dependencies?

Reply:
You can find example here https://github.com/wicketstuff/core/blob/master/select2-parent/select2/pom.xml#L41
I guess this way it will work and keep as small as possible :)

Reply:
Thanks. Refactored to use minimal dependencies.